core-bundle/src/Twig/Runtime/InsertTagRuntime.php

```diff
@@ -30,13 +30,25 @@ public function renderInsertTag(string $insertTag): string
         return $this->insertTagParser->renderTag($insertTag)->getValue();
     }
 
-    public function replaceInsertTags(string $text): string
+    public function replaceInsertTags(array $context, string $text, bool|null $bypass = null): string
```

Comment:
@m-vo Do you like `$bypass` or would you prefer `$keepInsertTags` as suggested by @ausi?

Reply:
I'm fine with both 😊